shoop/themes/classic_gray/templates/classic_gray/shoop/front/includes/language_changer.jinja

```diff
@@ -0,0 +1,19 @@
+{% if LANGUAGES|length > 1 %}
+<form method="post" action="/set-language/" class="hidden">
```

Comment:
Wouldn't that enable changing language of the admin interface from another site?